azurerm/internal/services/media/tests/media_asset_resource_test.go

```diff
@@ -0,0 +1,212 @@
+package tests
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/media/parse"
+)
+
+func TestAccAzureRMMediaAsset_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_media_asset", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMMediaAssetDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMMediaAsset_basic(data),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr(data.ResourceName, "description", "Asset description"),
+				),
+			},
+			data.ImportStep(),
+		},
+	})
+}
+
+func TestAccAzureRMMediaAsset_alternate_id(t *testing.T) {
```

Comment:
Unless they have to be seperate could we combine these into a complete test that sets them all, and then have an update test that goes basic -> complete -> basic ?